ClearentIdtechIOSFramework/Wrapper/ClearentWrapper.swift

```diff
@@ -208,6 +210,10 @@ public final class ClearentWrapper : NSObject {
                 let decodedResponse = try JSONDecoder().decode(TransactionResponse.self, from: responseData)
                 guard let transactionError = decodedResponse.payload.error else {
                     DispatchQueue.main.async {
+                        if let linksItem = decodedResponse.links?[0] {
```

Comment:
To make sure it doesn't crash if the array is empty, I would use:
```suggestion
                        if let linksItem = decodedResponse.links?.first {
```